packages/dashboard-frontend/src/pages/WorkspaceDetails/Header/Actions/index.tsx

```diff
@@ -74,7 +75,8 @@ export class HeaderActionSelect extends React.PureComponent<Props, State> {
       }
       this.props.history.push(nextPath);
     } catch (e) {
-      const message = `Unable to ${selected.toLocaleLowerCase()} ${this.props.workspaceName}. ` + e.toString().replace('Error: ', '');
+      const errorMessage = getErrorMessage(e);
+      const message = `Unable to ${selected.toLocaleLowerCase()} ${this.props.workspaceName}. ` + errorMessage.replace('Error: ', '');
```

Comment:
before I expanded it, it `selected` seems not very clear. Please consider renaming to something like `action`, `selectedAction`, `actionToPerform`

Reply:
yeah, it's a good time to make this clearer.

Reply:
+1 for  ```action```